src/util.php

```diff
@@ -54,8 +60,18 @@ public static function resolve_graphql_union_type($field)
         return $field['multiple'] ? ['list_of' => $union_name] : $union_name;
```

Comment:
Could this be a problem and it should also be a list if the field is either `multiple` or `cloned`?

Reply:
Yeah I missed this. Thanks! Will add a PR later today.

Reply:
#32